Calling/src/app/views/CallScreen.tsx

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { GroupCallLocator, TeamsMeetingLinkLocator } from '@azure/communication-calling';
+import { CommunicationUserIdentifier } from '@azure/communication-common';
+import {
+  CallAdapter,
+  CallAdapterState,
+  CallComposite,
+  createAzureCommunicationCallAdapter,
+  toFlatCommunicationIdentifier
+} from '@azure/communication-react';
+import { Spinner } from '@fluentui/react';
+import React, { useEffect, useRef, useState } from 'react';
+import { useSwitchableFluentTheme } from '../theming/SwitchableFluentThemeProvider';
+import { createAutoRefreshingCredential } from '../utils/credential';
+import MobileDetect from 'mobile-detect';
+
+const detectMobileSession = (): boolean => !!new MobileDetect(window.navigator.userAgent).mobile();
+
+export interface CallScreenProps {
+  token: string;
+  userId: CommunicationUserIdentifier;
+  callLocator: GroupCallLocator | TeamsMeetingLinkLocator;
+  displayName: string;
+  webAppTitle: string;
+  onCallEnded: () => void;
+}
+
+export const CallScreen = (props: CallScreenProps): JSX.Element => {
+  const { token, userId, callLocator, displayName, webAppTitle, onCallEnded } = props;
+  const [adapter, setAdapter] = useState<CallAdapter>();
+  const callIdRef = useRef<string>();
+  const adapterRef = useRef<CallAdapter>();
+  const { currentTheme, currentRtl } = useSwitchableFluentTheme();
+  const [isMobileSession, setIsMobileSession] = useState<boolean>(detectMobileSession());
+
+  useEffect(() => {
+    if (!callIdRef.current) {
+      return;
+    }
+    console.log(`Call Id: ${callIdRef.current}`);
+  }, [callIdRef.current]);
```

Comment:
Because callId is a ref is won't trigger a re-render and hence this console log won't be triggered when the callId ref changes (only once at the start)

Reply:
We only call `callIdRef.current =` in one place so perhaps we simplify and just have the console log there?

Reply:
Created as workitem
https://skype.visualstudio.com/SPOOL/_workitems/edit/2693381